src/modules/csm/state.py

```diff
@@ -166,3 +179,9 @@ def network_aggr(self) -> AttestationsAccumulator:
             included=included,
             assigned=assigned,
         )
+
+    def encode(self) -> bytes:
+        return StateJSONEncoder(indent=2).encode(self.dump()).encode()
```

Comment:
Do we need indent?

Reply:
I initially thought that it will be better to have a pretty printed JSON file. But I think we will go with zero instead. My test shows that for 1.6mln validators it's 114 MiB with indent=2 and 82 MiB with indent=0 which is significant change.